ARCdigi/include/ARCdigitizer.h

```diff
@@ -42,4 +48,12 @@ class DCHdigitizer : public GaudiAlgorithm {
   DataHandle<edm4hep::SimTrackerHitCollection> m_input_sim_hits{"inputSimHits", Gaudi::DataHandle::Reader, this};
   // Output digitized tracker hit collection name
   DataHandle<edm4hep::TrackerHitCollection> m_output_digi_hits{"outputDigiHits", Gaudi::DataHandle::Writer, this};
+  // Path to detector compact
+  StringProperty m_det_compact{this, "detectorCompact", ""};
```

Comment:
This should not need to be specified, the geometry is already built when calling the digitizer. You should do something like [this](https://github.com/HEP-FCC/k4SimGeant4/blob/acb6b90ab6b6b8dd5b9ce2b6864fe3bb43061187/Detector/DetComponents/src/GeoSvc.h#L46), instantiated with [this](https://github.com/HEP-FCC/k4SimGeant4/blob/acb6b90ab6b6b8dd5b9ce2b6864fe3bb43061187/Detector/DetComponents/src/GeoSvc.cpp#L65)

Reply:
Done - I think I have done this properly, but maybe you should double check...